pkg/util/node.go

```diff
@@ -28,7 +28,7 @@ func GetHostname(hostnameOverride string) string {
 	if string(hostname) == "" {
 		// Note: We use exec here instead of os.Hostname() because we
 		// want the FQDN, and this is the easiest way to get it.
```

Comment:
It looks like uname -n and os.Hostname() both eventually pull from "/proc/sys/kernel/hostname". I'm not sure if that is true for all distros.